qa/zcash/performance-measurements.sh

```diff
@@ -154,6 +154,13 @@ EOF
     xzcat block-107134.tar.xz | tar x -C "$DATADIR/regtest"
 }
 
+
+if [ $# -lt 2 ]
+then
+    echo "$0 : two arguments are required!"
```

Comment:
At least two arguments.

Reply:
Heh, of course it was possible to miss something in a trivial PR. I'll make a follow-up PR.

Reply:
@str4d already done 😸